src/ngraph/runtime/cpu/cpu_call_frame.cpp

```diff
@@ -79,6 +82,78 @@ void runtime::cpu::CPU_CallFrame::call(
     }
 }
 
+bool runtime::cpu::CPU_CallFrame::step(
+    const std::vector<std::shared_ptr<runtime::Tensor>>& output_tvs,
+    const std::vector<std::shared_ptr<runtime::Tensor>>& input_tvs)
+{
+    if (ctx->pc >= m_external_function->op_names.size())
+    {
+        return false;
+    }
+
+    ctx->breakpoints.insert(ctx->pc + 1);
+    inner_call(output_tvs, input_tvs);
+    ctx->breakpoints.erase(ctx->pc);
+    return true;
+}
+
+void runtime::cpu::CPU_CallFrame::kontinue(
```

Comment:
Is `kontinue` due to naming clash? 😆

Reply:
@ransford2011 but @diyessi gave me a great idea to rename it to `resume`